inc/sccm.class.php

```diff
@@ -428,7 +441,8 @@ function getMedias($deviceid, $limit = 99999999) {
       return $data;
    }
 
-   static function install() {
+   static function install()
+   {
```

Comment:
On update (so here) you should check if ```SCCMPush``` exist to delete if 

after updating the plugin this auto task is still present

![image](https://github.com/pluginsGLPI/sccm/assets/7335054/66dc4e2f-4617-4df4-9bf6-1c29a93ea491)